johnsonandjohnson/Bodiless-JS
# Feat (core, ui): Add remaining informed components

## Changes
- Add remaining elements from https://joepuzzo.github.io/informed/?path=/story/inputs--intro
- Add example 'form-elements' page

## Test Instructions
- Go to /form-elements page
- Make sure it follows the wireframe https://jhdfdc.axshare.com/#id=3ct389&p=aries

## Related Issues
Closes #336
